tests/core/daemon/test_daemon.py

```diff
@@ -69,16 +82,17 @@ async def num_connections():
 
         await time_out_assert_custom_interval(60, 1, num_connections, 1)
 
-        await time_out_assert(1500, node_height_at_least, True, node2, 1)
+        # await time_out_assert(60, node_height_at_least, True, node2, 1)
```

Comment:
@mariano54 Is this time out to wait until the system is up, or to check that we are advancing the chain?

Reply:
it waits for advancing the chain, checking that node2 has at least 2 blocks (height 1).